src/vm/CMakeLists.txt

```diff
@@ -520,6 +520,6 @@ convert_to_absolute_path(VM_SOURCES_DAC ${VM_SOURCES_DAC})
 add_subdirectory(dac)
 add_subdirectory(wks)
 
-if(CLR_CMAKE_PLATFORM_LINUX)
+if(CLR_CMAKE_PLATFORM_UNIX)
```

Comment:
Do you not need these on Window as well?

Reply:
No because this is needed only when the Generated code will be built external to the clr, which is on Unix. On Windows, the Generated folder will not be created